src/content/configuration/dev-server.md

```diff
@@ -804,7 +805,7 @@ T> The browser application name is platform dependent. Don't hard code it in reu
 
 ## `devServer.openPage`
 
-`string`
+`string` `array = []: string`
```

Comment:
Please see: https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types
```suggestion
`string` `[string]`
```